app/src/main/java/org/jellyfin/androidtv/ui/presentation/CardPresenter.java

```diff
@@ -393,7 +398,7 @@ public ViewHolder onCreateViewHolder(ViewGroup parent) {
         @ColorInt int color = typedValue.data;
         cardView.setBackgroundColor(color);
 
-        return new ViewHolder(cardView);
+        return new ViewHolder(cardView, ViewTreeLifecycleOwner.get(parent));
```

Comment:
Because our custom view is not connected to a parent we can't read the lifecycle from it, fortunately we know the parent  here so we can retrieve it from there.